### PR TITLE
Issue #2091: Move all user role settings to the user role settings page.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2091

### CODE REVIEW

File: core/modules/user/user.admin.inc [hunk 1]
```diff
@@ -531,6 +530,15 @@ function user_admin_roles($form, $form_state) {
  * Form submit function. Update the role weights.
  */
 function user_admin_roles_order_submit($form, &$form_state) {
+  // Save admin role and anon name.
+  dpm($form_state['values']);
```
Comment (quicksketch) [inline core/modules/user/user.admin.inc]: Heh. :wink:

**Comment by jenlampton** [comment]
removed dpm() and updated tests. Let's see if I got them all.

File: core/modules/user/user.admin.inc (hunk 1 above)
Comment (jenlampton) [reply core/modules/user/user.admin.inc]: _facepalm_

**Comment by quicksketch** [comment]
Looks great. Pending additional feedback: https://github.com/backdrop/backdrop-issues/issues/2091#issuecomment-242857493

**Comment by quicksketch** [comment]
Merged into 1.x for 1.5.0.

**Pull request merged**